src/OpenTelemetry.Resources.Host/HostResourceBuilderExtensions.cs

```diff
@@ -17,8 +17,19 @@ public static class HostResourceBuilderExtensions
     /// <param name="builder">The <see cref="ResourceBuilder"/> being configured.</param>
     /// <returns>The instance of <see cref="ResourceBuilder"/> being configured.</returns>
     public static ResourceBuilder AddHostDetector(this ResourceBuilder builder)
+    {
+        return AddHostDetector(builder, true);
```

Comment:
nit: Slightly better readability IMO:

```suggestion
        return AddHostDetector(builder, includeHostName: true);
```